examples/olmv1.yaml

```diff
@@ -0,0 +1,126 @@
+tests :
+  - name : olmv1-GCP
+    index: {{ es_metadata_index }}
+    benchmarkIndex: {{ es_benchmark_index }}
+    metadata:
+      platform: GCP
+      clusterType: self-managed
+      masterNodesType: e2-custom-6-16384
+      masterNodesCount: 3
+      workerNodesType: e2-standard-4
+      workerNodesCount: 3
+      benchmark.keyword: olm
+      ocpVersion: {{ version }}
+      not:
+        stream: okd
+    metrics : 
+    - name: catalogdMemory_GCP
+      metricName: catalogd_memory_peak_usage_5m
+      metric_of_interest: value
+      quantileName: Ready
+      not: 
+        jobConfig.name: "garbage-collection"
+      labels:
+        - "[Jira: OLMv1Performance]"
+    - name: controllerMemory_GCP
+      metricName: operator-controller_memory_peak_usage_5m
+      metric_of_interest: value
+      quantileName: Ready
+      not: 
+        jobConfig.name: "garbage-collection"
+      labels:
+        - "[Jira: OLMv1Performance]"
+    - name:  catalogdCPU_GCP
+      metricName : catalogd_cpu_usage_cores
+      metric_of_interest: value
+      labels.namespace.keyword: openshift-catalogd
+      not: 
+        jobConfig.name: "garbage-collection"
+      agg:
+        value: cpu
+        agg_type: avg
+      labels:
+        - "[Jira: OLMv1Performance]"
+    - name:  controllerCPU_GCP
+      metricName : operator-controller_cpu_usage_cores
+      metric_of_interest: value
+      labels.namespace.keyword: openshift-operator-controller
+      not: 
+        jobConfig.name: "garbage-collection"
+      agg:
+        value: cpu
+        agg_type: avg
+      labels:
+        - "[Jira: OLMv1Performance]"
+    - name: catalogdDisk_GCP
+      metricName: catalogd_disk_usage
+      metric_of_interest: value
+      quantileName: Ready
+      not: 
+        jobConfig.name: "garbage-collection"
+      labels:
+        - "[Jira: OLMv1Performance]"
+    - name: controllerDisk_GCP
+      metricName: controller_disk_usage
+      metric_of_interest: value
+      quantileName: Ready
+      not: 
+        jobConfig.name: "garbage-collection"
+      labels:
+        - "[Jira: OLMv1Performance]"
+    - name: catalogd_5xx_error_GCP
+      metricName: catalogd_http_5xx_error_count
+      metric_of_interest: value
+      quantileName: Ready
+      not: 
+        jobConfig.name: "garbage-collection"
+      labels:
+        - "[Jira: OLMv1Performance]"
```

Comment:
is this a proper jira project? if not, just remove this.

Reply:
Thanks! I updated it to an available one: `OpenShift Bugs`